pkgs/servers/lenpaste/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitea, git }:
+
+buildGoModule rec {
+  pname = "lenpaste";
+  version = "1.3";
+
+  src = fetchFromGitea {
+    domain = "git.lcomrade.su";
+    owner = "root";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-d+FjfEbInlxUllWIoVLwQRdRWjxBLTpNHYn+oYU3fBc=";
+  };
+
+  buildInputs = [ git ];
+
+  ldflags = "-w -s -X main.Version=${version}";
+
+  fixupPhase = ''
+    mv $out/bin/lenpaste-unwrapped $out/bin/lenpaste
+  '';
+
+  vendorSha256 = "sha256-PL0dysBn1+1BpZWFW/EUFJtqkabt+XN00YkAz8Yf2LQ=";
+
+  meta = with lib; {
```

Comment:
Please add `meta.maintainers` with at least one nixpkgs maintainer.
Consider adding yourself.
See https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers

Reply:
Thank you!